pkg/operator/controller/controller_dns_configmap.go

```diff
@@ -76,7 +76,7 @@ var corefileTemplate = template.Must(template.New("Corefile").Funcs(template.Fun
         {{- with $tls := .TransportConfig.TLS }}
         {{- with $serverName := $tls.ServerName }}
         tls_servername {{$serverName}}
-        tls {{- with $cm := $tls.CABundle.Name}} /etc/pki/{{$serverName}}-{{ index $.CABundleRevisionMap $cm }}/{{ $.CABundleFileName }}{{end}}
+        tls {{- with $cm := (index $.CABundleRevisionMap $tls.CABundle.Name) }} /etc/pki/{{$serverName}}-{{ $cm }}/{{ $.CABundleFileName }}{{end}}
```

Comment:
If it's possible for CABundleRevisionMap to be nil here, I think you have to also do a `{{- with .CABundleRevisionMap }}` check first.

Reply:
Sure, I've applied that.